angular2/README.md

```diff
@@ -1,4 +1,4 @@
-# Angular 2 Input Mask
+# Angular 2 / Ionic 2 Input Mask
 
```

Comment:
I think we should add a line here that says "This Angular 2 directive is also compatible with Ionic 2". And we should revert the other changes in this PR.

The reason I don't think the other changes are necessary is because I think most Ionic 2 users would expect an Angular 2 directive like this to work with Ionic 2, and the line above confirms that. On the other hand, a lot of Angular 2 users who view this page may not be interested in Ionic 2, and these mentions of Ionic 2 might be noisy or confusing.